templates/pulpcore-apache.conf.erb

```diff
@@ -0,0 +1,6 @@
+  ProxyPass <%= scope['pulpcore::apache::api_path'] %> <%= scope['pulpcore::apache::api_url'] %>
+  ProxyPassReverse <%= scope['pulpcore::apache::api_path'] %> <%= scope['pulpcore::apache::api_path'] %>
+  ProxyPassReverse <%= scope['pulpcore::apache::api_path'] %> <%= scope['pulpcore::apache::api_url'] %>
```

Comment:
Is the duplicate definition here intended?

Reply:
```
# grep -n Proxy /etc/httpd/conf.d/10-pulp.conf 
25:  ## Proxy rules
26:  ProxyRequests Off
27:  ProxyPreserveHost Off
28:  ProxyPass /pulp/api http://127.0.0.1:24817/pulp/api
29:  ProxyPassReverse /pulp/api /pulp/api
30:  ProxyPassReverse /pulp/api http://127.0.0.1:24817/pulp/api
31:  ProxyPass /pulp/content http://127.0.0.1:24816/pulp/content
32:  ProxyPassReverse /pulp/content /pulp/content
33:  ProxyPassReverse /pulp/content http://127.0.0.1:24816/pulp/content
```
is what is getting created by puppet-pulpcore itself as defined at https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/apache.pp#L16-L27

are the definitions there redundant as well?

Reply:
That's a good question and I think so. Looks to me like it should be `'reverse_urls' => [$api_url],` and probably also incorrect in `reverse_proxy.pp` in this module because I copied it from there.

Reply:
I've removed the redunant ProxyPassReverse declaration in all locations of which I am aware.